llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -509,19 +495,19 @@ static void getOperandsForBranch(Register CondReg, MachineRegisterInfo &MRI,
 bool RISCVInstructionSelector::select(MachineInstr &MI) {
   MachineBasicBlock &MBB = *MI.getParent();
   MachineFunction &MF = *MBB.getParent();
-  MachineRegisterInfo &MRI = MF.getRegInfo();
+  MRI = &MF.getRegInfo();
```

Comment:
This should go into a setupMF like other targets for function state setup

Reply:
Where other targets appears to just be AMDGPU and SPIRV?